### PR TITLE
Add dmaicher/doctrine-test-bundle

### PR DESCRIPTION
**Pull request by sstok**
| Q             | A
| ------------- | ---
| License       | MIT

@dmaicher can you check this and approve 👍

### CODE REVIEW

**Comment by sstok** [comment]
https://github.com/dmaicher/doctrine-test-bundle/pull/44 Should the reported violation.

**Comment by stof** [comment]
Looks like the merged PR does not fix the validation.

@fabpot does the validator require to have it in a release ?

File: dama/doctrine-test-bundle/4.0/config/packages/dmaicher_doctrine_test_bundle.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+dama_doctrine_test:
```
Comment (fabpot) [inline dama/doctrine-test-bundle/4.0/config/packages/dmaicher_doctrine_test_bundle.yaml]: This should be moved to `packages/test/` as the bundle is only available in the test environment.

**Comment by dmaicher** [comment]
@sstok can you do the required change? :blush:

**Comment by dmaicher** [comment]
@fabpot @sstok this PR can be closed. See https://github.com/symfony/recipes-contrib/pull/171 instead.

**Pull request closed without merging**